clang/include/clang/Basic/DarwinSDKInfo.h

```diff
@@ -181,6 +192,7 @@ class DarwinSDKInfo {
   VersionTuple Version;
   VersionTuple MaximumDeploymentTarget;
   llvm::Triple::OSType OS;
+  std::unordered_map<llvm::Triple, std::string> SystemPrefixes;
```

Comment:
I'm not really sure if this makes better sense as a `SmallDenseMap` or an `unordered_map`. I tried both and they both work. `unordered_map` is a little less code in `Triple` to make it work, and @felipepiovezan suggested that was better.